### PR TITLE
Make it possible to use a non-standard SSH port, allow for deploys to…

### PR DESCRIPTION
**Pull request by robottaway**
… pd-lite env

### CODE REVIEW

File: lib/pd-cap-recipes/util.rb [hunk 1]
```diff
@@ -37,9 +37,9 @@ def collect_version_info(servers, user, gateway=nil)
 end
 
 # Allow passing a block to process on each cluster/stage machine
-def  ssh_to_cluster(servers, user, gateway_host=nil)
+def ssh_to_cluster(servers, user, port, gateway_host: nil)
   if gateway_host != nil
-    gateway = Net::SSH::Gateway.new(gateway_host, user)
+    gateway = Net::SSH::Gateway.new(gateway_host, user, :port => port)
```
Comment (margueritepd) [inline lib/pd-cap-recipes/util.rb]: Could use `port: port` syntax

File: lib/pd-cap-recipes/util.rb [hunk 2]
```diff
@@ -17,9 +17,9 @@ def get_current_rev_info(servers, user, gateway=nil)
 end
 
 # Return a hash with host -> hash of information on release folder mapping
-def collect_version_info(servers, user, gateway=nil)
+def collect_version_info(servers, user, port, gateway_host: nil)
```
Comment (margueritepd) [inline lib/pd-cap-recipes/util.rb]: Looks like `collect_version_info` and `get_current_rev_info` always has `gateway_host` passed to it. Any reason to keep this an optional argument?

**Comment by margueritepd** [comment]
Seems reasonable :wave:

File: lib/pd-cap-recipes/util.rb (hunk 2 above)
Comment (robottaway) [inline lib/pd-cap-recipes/util.rb]: More future proofing than anything.

File: lib/pd-cap-recipes/util.rb (hunk 1 above)
Comment (robottaway) [inline lib/pd-cap-recipes/util.rb]: Yeah I actually left it that way just to be consistent with the net::ssh docs. Might be silly idk.

**Pull request merged**